src/Microsoft.Graph.Core/Requests/DeltaResponseHandler.cs

```diff
@@ -20,16 +22,16 @@ namespace Microsoft.Graph
     /// deserializer can't express changes to null so you can now discover if a property
     /// has been set to null. This is intended for use with a Delta query scenario.
     /// </summary>
-    public class DeltaResponseHandler : IResponseHandler
+    public class DeltaResponseHandler<T> : IResponseHandler where T : IParsable
     {
-        private readonly ISerializer serializer;
+        private readonly JsonParseNodeFactory parseNodeFactory;
 
         /// <summary>
-        /// Constructs a new <see cref="ResponseHandler"/>.
+        /// Constructs a new <see cref="DeltaResponseHandler{T}"/>.
         /// </summary>
         public DeltaResponseHandler()
         {
-            this.serializer = new Serializer();
+            this.parseNodeFactory = new JsonParseNodeFactory();
```

Comment:
QQ: why are you hardcoding a new instance instead of relying on the default factory instance to get you a new one?

Reply:
Thanks for pointing this out. It's much better that way with the added advantage of the user configuring their own. 
This is addressed in the latest update.